docs/articles/testkube-dashboard-workflows-templates.md

```diff
@@ -1,102 +1,102 @@
 
 :::
 
-## Adding a New Workflow Template
+## Add a New Template
 
-Click the **Add a new template** button in the Workflows Template overview (shown above) to create a new Workflow Template. 
-The options are:
+![Workflow Templates](../articles/images/workflow-templates-add-new-template.png)
 
-- **Git content**: Create a template to use a git source across multiple test workflows.
-- **Static content**: Create a template to use a static file across multiple test workflows.
-- **Testing tool**: Create a template with a
-container/runnable testing tool across to base your test workflows on.
-- **Import from YAML**: Create a template from scratch or copy/paste YAML examples.
+Select **Add a new template** to create **Workflow Templates** specific to:
 
-![Add a New Template](../img/add-new-workflow-template-072024.png)
+* **Git Content** – Create a template to use a Git Source across multiple **Test Workflows**.
+* **Static Content** – Create a template to use a static file across multiple **Test Workflows**.
+* **Testing Tool** – Create a template with a container/runnable testing tool on which to base your **Test Workflows**.
+* **Import from YAML** – Create a template from scratch or copy/paste YAML examples.
 
-### Creating a Git Content Template
+### Git Content
 
-The Git Content template wizard helps you create a Workflow Template that you can use across your workflows to ensure
-they are all using the same Git Source as their content.
+You can create a template to use across Workflows, ensuring each uses the same Git Source as its content. For example, all Workflows might use the same Git repository/branch or even the same access method
+such as SSH or HTTPS.
 
-![Git Workflow Template](../img/git-content-template-wizard.png)
+![Git Content Template](../articles/images/workflow-templates-git-content.png)
 
 
-### Creating a Static Content Template
+### Static Content
 
-The Static content template wizard helps you create a Workflow Template that you can use across your workflows to ensure 
-they are all using the same static content in corresponding scenarios.
+You can create a template to use across Workflows, ensuring each uses the same static content for the same scenarios. For example, Deployment Workflows might use the same template such that
+the production setup text is identical. Similarly, Onboarding Workflows might reference the same static instructions for new users.
 
-![Static Content Template Wizard](../img/static-content-template-wizard.png)
+![Static Content Template](../articles/images/workflow-templates-static-content.png)
 
 
-### Creating a Testing Tool Template
+### Testing Tool
 
-The Testing Tool template wizard helps you create a Workflow Template that you can use across your workflows to ensure
-they are all using a specific testing tool or script in a consistent way.
+You can create a template to use across Workflows such that each uses a specific testing tool or script the same way.
```

Comment:
"This option creates a WorkflowTemplate that ensures your Workflows use the same testing tool or.."